### PR TITLE
update header and theme

### PR DESCRIPTION
**Pull request by frzyc**
## Issue or discord link

- 

## Testing/validation

New(left) vs. Old(right)
Screenshot 2024-06-23 001905

Mobile
image

## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)

- [ ] I have commented my code in hard-to understand areas.
- [ ] I have made corresponding changes to README or wiki.
- [ ] For front-end changes, I have updated the corresponding English translations.
- [ ] I have run `yarn run mini-ci` locally to validate format and lint.
- [ ] If I have added a new library or app, I have updated the deployment scripts to ignore changes as needed

### CODE REVIEW

File: apps/frontend/src/app/Header.tsx [hunk 1]
```diff
@@ -217,37 +220,34 @@ function HeaderContent({ anchor }: { anchor: string }) {
               transition: 'background-color 0.5s ease',
               backgroundColor: 'rgba(255,255,255,0.1)',
             },
-          }}
+            '& .Mui-selected': {
```
Comment (nguyentvan7) [inline apps/frontend/src/app/Header.tsx:223]: I think the text is supposed to be bolded as well
design doc:
image

site:
image

Comment (frzyc) [reply apps/frontend/src/app/Header.tsx:223]: I did try to do that, but bolding the text makes it slightly wider, so it causes a tiny bit of layout shift

Comment (nguyentvan7) [reply apps/frontend/src/app/Header.tsx:223]: Can you try with pseudoelement with prebolded text? I think the current design is a bit hard to notice which tab you're on even with the colorization

https://stackoverflow.com/questions/556153/inline-elements-shifting-when-made-bold-on-hover

Comment (frzyc) [reply apps/frontend/src/app/Header.tsx:223]: I've tried that, and many other solutions like fake bolding with dropshadow and manual letter-spacing. I've not gotten it to work and I don't want to spend more time on it.
also, you can't seriously argue that you can't see which tab is activated here if the text isn't bolded?

image

**Review by nguyentvan7 — APPROVED** [review]
/shrug

**Pull request merged**